alephium/contracts/sequence/undone_sequence.ral

```diff
@@ -0,0 +1,26 @@
+Contract UndoneSequence(
+    parentId: ByteVec,
+    begin: U256,
+    mut sequences: U256,
+    refundAddress: Address
```

Comment:
Would it make sense to have `refundAddress` being the address of the `Sequence` contract (or its subclass)? Since `Sequence` contract is the one that spent the ALPH and created the `UndoneSequence`, I guess if the money goes back to it when all sequences in `UndoneSequence` are checked, it will reduce the amount of manual intervention.

Reply:
It would be great if the `refundAddress` is the `Sequence` contract, but the `destroySelf` only supports asset addresses.